### PR TITLE
Fetch reps profiles from ReMo.

### LINKED ISSUE #50: Add reps profiles in app.

### PR DESCRIPTION
**Pull request by akatsoulas**
Fixes #50

### CODE REVIEW

**Comment by johngian** [comment]
Code looks OK. Tested it locally and returns some profiles as expected.
r+

File: woodstock/voting/utils.py [hunk 1]
```diff
@@ -13,11 +13,11 @@
...
 
 API_RESULTS_LIMIT = 200
-MOZILLIANS_API_URL = settings.MOZILLIANS_API_URL
+REPS_PROFILE_URL = 'https://reps.moziilla.org/u/'
 
 
-def query_mozillians_api(url):
-    """Helper method to query Mozillians.org API"""
+def query_api(url):
+    """Helper method to query an API"""
```
Comment (johngian) [inline woodstock/voting/utils.py]: Please change: to query mozillians/reps API

File: woodstock/voting/utils.py [hunk 2]
```diff
@@ -13,11 +13,11 @@
 
 
 API_RESULTS_LIMIT = 200
-MOZILLIANS_API_URL = settings.MOZILLIANS_API_URL
+REPS_PROFILE_URL = 'https://reps.moziilla.org/u/'
```
Comment (johngian) [inline woodstock/voting/utils.py]: Fix typo

Comment (johngian) [inline woodstock/voting/utils.py]: Or better move this to settings.

File: woodstock/voting/utils.py [hunk 3]
```diff
@@ -39,8 +39,19 @@ def get_mozillian_by_email(email):
...
+    data = {
+        'limit': API_RESULTS_LIMIT,
+        'query': username
+    }
+
+    url = settings.REPS_API_URL + '?format=json&' + urllib.urlencode(data)
+    return query_api(url)
 
 
 def get_mozillian_by_username(username):
```
Comment (johngian) [inline woodstock/voting/utils.py:57]: Please fix the docstring here as you did to `get mozillians by email`.

**Comment by akatsoulas** [comment]
@johngian thanks for the review!

**Pull request merged**